tests/test-runner/bin/zts-report.py

```diff
@@ -260,6 +260,8 @@
         'cli_root/zpool_expand/zpool_expand_001_pos': ['FAIL', known_reason],
         'cli_root/zpool_expand/zpool_expand_005_pos': ['FAIL', known_reason],
         'cli_root/zpool_reopen/zpool_reopen_003_pos': ['FAIL', known_reason],
+        'limits/filesystem_limit': ['FAIL', '8226'],
+        'limits/snapshot_limit': ['FAIL', '8226'],
```

Comment:
It's too bad that we will be masking future regressions of these tests, especially since our automated tests (which don't yet run on any old kernels) don't fail.  I wonder if the tests could do a runtime check for HAVE_HAS_CAPABILITY and if not then bail out?

Reply:
It's unfortunate, but not quite as bad as it may initially seem.  The exception here is specific to Linux, so any regressions in the common code will be caught by the FreeBSD builders.  A run time check would be ideal.  However it's problematic since there's not an easy way to check if the kmods were built with HAVE_HAS_CAPABILITY from the test suite.  Checking the kernel version isn't helpful either since we know some older kernels have the patch applied.  Without adding some kind of new infrastructure to support these kinds of check this is what we can do today.

Reply:
Couldn't we run these tests on all kernels >=4.10?

Reply:
Maybe SKIP would be a bit nicer than maybe FAIL.

Reply:
Sure, we can do that.  I've updated the PR to skip the tests on <4.10 kernels and to expect either a PASS or SKIP result.